paasta_tools/instance/kubernetes.py

```diff
@@ -867,3 +867,66 @@ def ready_replicas_from_replicaset(replicaset: V1ReplicaSet) -> int:
         ready_replicas = 0
 
     return ready_replicas
+
+
+def kubernetes_mesh_status(
+    service: str,
+    instance: str,
+    verbose: int,
+    instance_type: str,
+    settings: Any,
+    include_smartstack: bool = True,
+    include_envoy: bool = True,
+) -> Mapping[str, Any]:
+
+    if not include_smartstack and not include_envoy:
+        return {}
+    if instance_type not in LONG_RUNNING_INSTANCE_TYPE_HANDLERS:
+        return {}
+
+    config_loader = LONG_RUNNING_INSTANCE_TYPE_HANDLERS[instance_type].loader
+    job_config = config_loader(
+        service=service,
+        instance=instance,
+        cluster=settings.cluster,
+        soa_dir=settings.soa_dir,
+        load_deployments=True,
+    )
+    service_namespace_config = kubernetes_tools.load_service_namespace_config(
+        service=service,
+        namespace=job_config.get_nerve_namespace(),
+        soa_dir=settings.soa_dir,
+    )
+    if "proxy_port" not in service_namespace_config:
+        return {}
+
+    kube_client = settings.kubernetes_client
+    if kube_client is None:
+        return {}
+    pod_list = kubernetes_tools.pods_for_service_instance(
+        service=job_config.service,
+        instance=job_config.instance,
+        kube_client=kube_client,
+        namespace=job_config.get_kubernetes_namespace(),
+    )
+
+    kmesh: Dict[str, Any] = {}
+    mesh_status_kwargs = dict(
+        service=service,
+        instance=job_config.get_nerve_namespace(),
+        job_config=job_config,
+        service_namespace_config=service_namespace_config,
+        pods=pod_list,
+        should_return_individual_backends=verbose > 0,
```

Comment:
I kind of vote that we make this always True -- since the idea is that it'd only be run when we need more info on a service instance's backends, seems like we will probably almost always want the full table.

Reply:
i removed the verbose arg and made this always true